x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -159,10 +135,13 @@ cluster.
 ----
 bin/elasticsearch
 ----
++
```

Comment:
The `Enroll additional nodes in your cluster` section is outdated. Users don't need to run `elasticsearch-enroll-node --enrollment-token <token>` before they start the node, in fact that CLI tool is not available any more. 

The just need to start the additional node the first time with `bin/elasticsearch --enrollment-token <token>` 

And the path is `tls_auto_config_` (wihtout `initial` )

Reply:
Thanks for pointing this out -- I'll remove the unnecessary step and clarify that directory name.

Reply:
This command is now updated via 33e73ecc1d4a2a7a992b97d66d854c06b959093a. Missed it in my previous push.